cloudbaseinit/plugins/common/sshpublickeys.py

```diff
@@ -34,7 +34,7 @@ def execute(self, service, shared_data):
             LOG.debug('Public keys not found in metadata')
             return base.PLUGIN_EXECUTION_DONE, False
 
-        username = CONF.username
```

Comment:
this change should be removed from this patch, I will add another patch to have this fix merged asap.